src/app/components/list/view-switcher/index.html

```diff
@@ -21,44 +21,44 @@
       <h3 class="sky-text-success">Use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Use the list view switcher when the user needs to switch between different views of the same data.</p>
+              <p>Use list view switchers when users need to switch between different views of the same data.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/use-when.png" />
-              <div class="caption caption-success sky-text-success">Do use the list view switcher when there is more than one view of the data.</div>
+              <div class="caption caption-success sky-text-success">Do use list view switchers when more than one view of data is available to users.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
-      <h3 class="sky-text-success">Don't use when</h3>
+      <h3 class="sky-text-danger">Don't use when</h3>
         <sky-docs-visual-example>
           <sky-docs-visual-example-text>
-              <p>Don't use the list view switcher when there is only one view of the data.</p>
+              <p>Don't use list view switchers when only one view of data is available.</p>
           </sky-docs-visual-example-text>
           <sky-docs-visual-example-image>
             <div>
               <img src="~/assets/img/guidelines/list-view-switcher/dont-use-when.png" />
-              <div class="caption caption-danger sky-text-danger">Don't use the list view switcher when there is only one view of the data.</div>
+              <div class="caption caption-danger sky-text-danger">Don't use list view switchers when users cannot switch between different views.</div>
             </div>
           </sky-docs-visual-example-image>
         </sky-docs-visual-example>
     </sky-demo-page-content>
 
     <sky-demo-page-content sectionHeading="Anatomy">
```

Comment:
@Blackbaud-ToddRoberts, it might just be me, but I was initially slightly confused by the icons and the active/inactive labels here. Since the icons here are the same as the icons in the other images, I thought for a minute that the first item here was a default button choice for displaying an "active view" and that the second one was a default button choice for displaying an "inactive view." I'm not sure if there's anything that needs to be addressed for this, but I just wanted to mention it in case you think other users might initially be similarly confused. The only tweak I can think of is to maybe use different icons in at least one of the Use when/Don't use when images. But I'm not sure if that's necessary; just throwing it out there.

Reply:
@Blackbaud-ToddRoberts, I assume that we don't need to worry about this?

Reply:
i think it's ok, we can update the image later if it is causing confusion.

Reply:
Sounds good. Resolving this comment...